src/App.tsx

```diff
@@ -1,14 +1,45 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { useAppSelector } from './app/hooks';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { actions as todosAction } from './features/todos';
+import { filterAndSortTodos } from './services/Filter';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const dispatch = useDispatch();
+  const todos = useAppSelector(state => state.todos);
+  const selectedTodo = useAppSelector(state => state.currentTodo);
+  const { query, status } = useAppSelector(state => state.filter);
+
+  const [loading, setLoading] = useState(false);
```

Comment:
Rename all boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isLooading, setIsLoading] = useState(false);
```